### PR TITLE
Added site with cert-based auth

### PR DESCRIPTION
**Pull request by mikeliddle**

### CODE REVIEW

File: CreateServer.ps1 [hunk 1]
```diff
@@ -196,8 +196,9 @@ Function Test-Prerequisites {
     
     if (-Not (Get-Module -ListAvailable -Name "Microsoft.Graph")) {
         Write-Header "Installing Microsoft.Graph..."
-        Install-Module Microsoft.Graph -Force
+        Install-Module Microsoft.Graph -Force -RequiredVersion 1.28.0
```
Comment (toddbohm) [inline CreateServer.ps1:199]: I assume we would want "MinimumVersion"

File: nginx_data/trusted.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <html>
         <H1>Welcome to the Trusted Site</H1>
-        <H2>This is secured by a private TLS Cert</H2>
+        <H2>This is secured using TLS Mutual Authentication</H2>
```
Comment (toddbohm) [inline nginx_data/trusted.html:3]: Don't we want two sites? One with client certs and one without?

I would also recommend a third. One with optional client certificates.

**Review by toddbohm — CHANGES_REQUESTED** [review]

File: CreateServer.ps1 (hunk 1 above)
Comment (mikeliddle) [reply CreateServer.ps1:199]: 1.28.0 is the Last version of v1, and v2 requires some significant updates before we can upgrade to that. So I went with required so we'd stay below v2 until we're ready to upgrade.

File: nginx_data/trusted.html (hunk 2 above)
Comment (mikeliddle) [reply nginx_data/trusted.html:3]: I'll add an item to create the other site, the trusted.html site was removed, so I just repurposed that file to be our certificate site.

**Comment by mikeliddle** [comment]
Added #85 To address the optional certificate site request

**Comment by mikeliddle** [comment]
Added #86 to update to v2 of the graph cmdlets.

**Review by toddbohm — APPROVED** [review]

**Pull request merged**